tests/ui/test_install.py

```diff
@@ -0,0 +1,20 @@
+import pytest
+
+
+from pages.desktop.details import Details
+
+
+@pytest.mark.nondestructive
+def test_addon_install(
+        base_url, selenium, firefox, firefox_notifications):
+    """Test that navigates to an addon and installs it."""
+    selenium.get('{}/firefox/addon/ui-test-install'.format(base_url))
+    addon = Details(selenium, base_url)
+    assert 'Ui-Addon-Install' in addon.description_header.name
+    addon.description_header.install_button.click()
+    firefox.browser.wait_for_notification(
+        firefox_notifications.AddOnInstallBlocked).allow()
+    firefox.browser.wait_for_notification(
+        firefox_notifications.AddOnInstallConfirmation).install()
+    firefox.browser.wait_for_notification(
+        firefox_notifications.AddOnInstallComplete).close()
```

Comment:
That's nice! Well, that doesn't need documentation anymore, glorious. Just curious, does `wait_for_notification` imply a timeout?

Reply:
Yes a 10 second timeout for the notification to appear.